frontend/src/metabase/redux/uploads.ts

```diff
@@ -62,7 +62,7 @@ export const uploadFile = createThunkAction(
         dispatch(
           uploadError({
             id,
-            message: t`You cannot upload files larger than ${MAX_UPLOAD_STRING}`,
+            message: t`You cannot upload files larger than ${MAX_UPLOAD_STRING}mb`,
```

Comment:
Should we keep the `mb` as part `MAX_UPLOAD_STRING`? That way the string doesn't have to be re-translated if we change the limit in the future to the order of gigabytes. Also, translation strings may or may not have been frozen yet for 47, not entirely sure.

Reply:
Ah nvm I just saw Tim's comment. I think it is better to keep `mb` outside of `MAX_UPLOAD_STRING` since we want to it to be translate-able.